### PR TITLE
git.ignore cleanup

### PR DESCRIPTION
**Pull request by amznestebanpapp**
Signed-off-by: Esteban Papp <81431996+amznestebanpapp@users.noreply.github.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,18 +2,14 @@
 .vs/
 .vscode/
 __pycache__
-AssetProcessorTemp/**
```
Comment (amznestebanpapp) [inline .gitignore:5]: this was moved to the cache folder

File: .gitignore [hunk 2]
```diff
@@ -2,18 +2,14 @@
...
 [Bb]uild/
 [Oo]ut/**
 CMakeUserPresets.json
 [Cc]ache/
 /[Ii]nstall/
-Editor/EditorEventLog.xml
-Editor/EditorLayout.xml
```
Comment (amznestebanpapp) [inline .gitignore:12]: these were moved into the user folder

File: .gitignore [hunk 3]
```diff
@@ -2,18 +2,14 @@
...
 **/*egg-info/**
 **/*egg-link
 **/[Rr]estricted
-UserSettings.xml
```
Comment (amznestebanpapp) [inline .gitignore:16]: moved to the User folder

File: .gitignore [hunk 4]
```diff
@@ -22,9 +18,6 @@ client*.cfg
 server*.cfg
 .mayaSwatches/
 _savebackup/
-#Output folder for test results when running Automated Tests
-TestResults/**
```
Comment (amznestebanpapp) [inline .gitignore:26]: this is in the binary output folder (build folder)

File: .gitignore [hunk 5]
```diff
@@ -22,9 +18,6 @@ client*.cfg
...
 *.swatches
 /imgui.ini
-/scripts/project_manager/logs/
```
Comment (amznestebanpapp) [inline .gitignore:29]: couldnt find any code writing to `project_manager`

File: .gitignore [hunk 6]
```diff
@@ -22,9 +18,6 @@ client*.cfg
...
-/AutomatedTesting/Gem/PythonTests/scripting/TestResults
```
Comment (amznestebanpapp) [inline .gitignore:30]: this is outputting to the output folder (build folder)

**Review by lumberyard-employee-dm — APPROVED** [review]

**Review by brianherrera — APPROVED** [review]

**Review by AMZN-alexpete — APPROVED** [review]

**Review by amzn-changml — APPROVED** [review]

**Pull request merged**